src/database/utilities/createRecord.js

```diff
@@ -206,6 +210,24 @@ function createStocktakeBatch(database, stocktakeItem, itemBatch) {
   return stocktakeBatch;
 }
 
+// Creates a supplier invoice (Transaction) and adds it to the customer (Name)
+function createSupplierInvoice(database, supplier, user) {
+  const currentDate = new Date();
+  const invoice = database.create('Transaction', {
+    id: generateUUID(),
+    serialNumber: getNextNumber(database, SUPPLIER_INVOICE_NUMBER),
```

Comment:
**Important** Have you talked with others about how locally assigned supplier invoice numbers avoid conflicts with those that are centrally assigned?